mmomni/ansible/playbooks/jitsi/jitsi-config.js

```diff
@@ -28,10 +28,10 @@ var config = {
     },
 
     // BOSH URL. FIXME: use XEP-0156 to discover it.
-    bosh: '//{{ jitsi_fqdn }}/http-bind',
+    // bosh: '//{{ jitsi_fqdn }}/http-bind',
```

Comment:
Mobile devices still use bosh, leave it available :-)

Reply:
Done. Thanks!